### PR TITLE
[🧹 UNIFICATION CLEANUP] Separate out `admin.subscription.v1` feature

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

Move `extensions > components > subscription` to `admin.subscription.v1`

### Related Issues
- https://github.com/wso2/product-is/issues/20802

### Related PRs

- N/A

### Checklist
- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

File: .changeset/warm-roses-lay.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.core.v1": minor
+"@wso2is/console": minor
```
Comment (pavinduLakshan) [inline .changeset/warm-roses-lay.md:3]: We do not add a new feature to console, don't we?

**Review by pavinduLakshan — DISMISSED** [review]

File: .changeset/warm-roses-lay.md (hunk 1 above)
Comment (brionmario) [reply .changeset/warm-roses-lay.md:3]: We have removed the feature from extensions and added it as a separate feature in Console.

That said, any improvements, refactoring should go in a `minor` release.

Only bug fixes can go in `patch`.

File: features/admin.subscription.v1/contexts/subscription-context.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (dasuni-30) [inline features/admin.subscription.v1/contexts/subscription-context.ts:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: features/admin.subscription.v1/hooks/use-subscription.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (dasuni-30) [inline features/admin.subscription.v1/hooks/use-subscription.ts:2]:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

File: features/admin.subscription.v1/public-api.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```
Comment (dasuni-30) [inline features/admin.subscription.v1/public-api.ts:2]:
```suggestion
 * Copyright (c) 2023-2024, WSO2 LLC. (https://www.wso2.com).
```

**Review by dasuni-30 — DISMISSED** [review]

**Review by dasuni-30 — APPROVED** [review]

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**